src/build/build.ts

```diff
@@ -26,6 +24,10 @@ import {PrefetchTransform} from './prefetch';
 import {waitFor} from './streams';
 import {parsePreCacheConfig} from './sw-precache';
 
+// TODO(fks) 09-22-2016: Latest npm type declaration resolves to a non-module
+// entity. Upgrade to import once compatible .d.ts file is released.
+import mergeStream = require('merge-stream');
```

Comment:
Is that a problem? I'm pretty sure `import * as mergeStream from 'merge-stream'` works with non-module entities (even if it shouldn't)